src/scanner.c

```diff
@@ -135,9 +139,11 @@ void            SyntaxError (
         Pr( "%s", (Int)STATE(Input)->line, 0L );
 
         /* print a '^' pointing to the current position                        */
-        for ( i = 0; i < STATE(In) - STATE(Input)->line - 1; i++ ) {
-          if ( STATE(Input)->line[i] == '\t' )  Pr("\t",0L,0L);
-          else  Pr(" ",0L,0L);
+        Int pos = GetLinePosition();
+        for (i = 0; i < pos; i++) {
+            if (STATE(Input)->line[i] == '\t')
+                Pr("\t", 0L, 0L);
+            else  Pr(" ",0L,0L);
```

Comment:
Did you run `git clang-format` on this? Just wondering, because `Pr` is in the same line as `else`.

Reply:
(since the rest of the function is clearly not clang formatted, it's also fine to leave it like this. I was simply wondering).